demonstrations/tutorial_qsvt_hardware.py

```diff
@@ -0,0 +1,161 @@
+
+obs = [qml.X(3), qml.X(3) @ qml.Z(4), qml.Z(3) @ qml.Y(4)]
+
+H = qml.dot(coeffs, obs)
+
+H_mat = qml.matrix(H, wire_order=[3, 4])
+
+# We calculate p(H) = -H + 0.5 * H^3 + 0.5 * H^5
+H_poly = -H_mat + 0.5 * mpow(H_mat, 3) + 0.5 * mpow(H_mat, 5)
+
+print(np.round(H_poly, 4))
+
+######################################################################
+# Now that we know what the target result is, let's see how to apply the polynomial with a quantum circuit instead.
+# We start by defining the proper input operators for the :class:`~.qml.QSVT` template.
+
+# We need |log2(len(coeffs))| = 2 control wires to encode the Hamiltonian
+control_wires = [1, 2]
+block_encode = qml.PrepSelPrep(H, control=control_wires)
+
+projectors = [
+    qml.PCPhase(angles[i], dim=2 ** len(H.wires), wires=control_wires + H.wires)
+    for i in range(len(angles))
+]
+
+
+dev = qml.device("default.qubit")
+
+@qml.qnode(dev)
+def circuit():
+
+    qml.Hadamard(0)
+    qml.ctrl(qml.QSVT, control=0, control_values=[1])(block_encode, projectors)
+    qml.ctrl(qml.adjoint(qml.QSVT), control=0, control_values=[0])(block_encode, projectors)
+    qml.Hadamard(0)
+
+    return qml.state()
+
+
+matrix = qml.matrix(circuit, wire_order=[0] + control_wires + H.wires)()
+print(np.round(matrix[: 2 ** len(H.wires), : 2 ** len(H.wires)], 4))
+
+######################################################################
+# The matrix obtained using QSVT is the same as the one obtained by applying the polynomial
+# directly to the Hamiltonian! That means the circuit is encoding :math:`p(\mathcal{H})` correctly.
+# The great advantage of this approach is that all the building blocks used in the circuit can be
+# decomposed into basic gates easily, allowing this circuit
+# to be easily executed on hardware devices with PennyLane.
+#
+# Please also note that QSVT encodes the desired polynomial :math:`p(\mathcal{H})` as well as
+# a polynomial :math:`i q(\mathcal{H})`. To isolate :math:`p(\mathcal{H})`, we have used an auxiliary qubit and considered that
+# the sum of a complex number and its conjugate gives us twice its real part. We
+# recommend :doc:`this demo </demos/tutorial_apply_qsvt>` to learn more about the structure
+# of the circuit.
+#
+# Conclusion
+# ----------
+# In this brief how-to we demonstrated applying QSVT on a sample Hamiltonian. Note that the algorithm is sensitive to
+# the block-encoding method, so please make sure that the projection angles are converted to the proper format.
+# This how-to serves as a guide for running your own workflows and experimenting with more advanced Hamiltonians and polynomial functions.
+#
+# References
+# ----------
+#
+# .. [#unification]
+#
+#     John M. Martyn, Zane M. Rossi, Andrew K. Tan, Isaac L. Chuang.
+#     "A Grand Unification of Quantum Algorithms".
+#     `arXiv preprint arXiv:2105.02859 <https://arxiv.org/abs/2105.02859>`__.
+#
+# About the author
+# ----------------
+#
```

Comment:
To fix the rendering issue:
```suggestion
#

```